src/test/java/com/squareup/javapoet/MethodSpecTest.java

```diff
@@ -39,8 +48,25 @@
 public final class MethodSpecTest {
   @Rule public final CompilationRule compilation = new CompilationRule();
 
+  private Elements elements;
+  private Types types;
+
+  @Before
+  public void init() {
```

Comment:
And a convention is a convention is a convention. Contract. Consider it renamed.